spec/index.html

```diff
@@ -1125,18 +1125,27 @@ <h3>Reifying Statements: <code>rdf:ID</code></h3>
   </dl>
 
   <section id="section-MIME-Type">
-    <h2>RDF MIME Type, File Extension and Macintosh File Type</h2>
+    <h2>RDF/XML Internet Media Type, File Extension and Macintosh File Type</h2>
+    <dl>
+      <dt>Contact:</dt>
+      <dd>Dan Brickley</dd>
+      <dt>See also:</dt>
 
-    <p>The Internet media type / MIME type for RDF/XML is
-    <code>application/rdf+xml</code> &mdash; [[RFC3870]].
-    </p>
+      <dd><a href="http://www.w3.org/2002/06/registering-mediatype">How to Register a Media Type for a W3C Specification</a></dd>
+      <dd><a href="http://www.w3.org/2001/tag/2002/0129-mime">Internet Media Type registration, consistency of use</a><br />TAG Finding 3 June 2002 (Revised 4 September 2002)</dd>
+    </dl>
 
-    <p>It is recommended that RDF/XML files have the extension
-    <code>".rdf"</code> (all lowercase) on all platforms.</p>
+    <p>The Internet Media Type / MIME type for RDF/XML is
+      <code>application/rdf+xml</code> &mdash; [[RFC3870]].</p>
+
+    <p>It is recommended that RDF/XML files have the extension &quot;.rdf&quot;
+      (all lowercase) on all platforms.</p>
```

Comment:
This would be a big change; even though it may not have been well-considered to use the ".rdf" extension, it is what it is, and I don't know that we want to make that change, and update the corresponding RFC, without due consideration.

Reply:
> Wording that implies how people think is not appropriate for a spec

I welcome suggestion of revised wording that conveys the fact of the common misunderstanding that RDF was inherently XML, due in no small part to the historical recommendation of the `.rdf` filename extension for RDF/XML documents (and also to there being no RDF 1.0 examples in any other serialization).  The quoted phrase is from a sentence which belongs here, with discussion of the Media Type and filename extension.

Reply:
I've revised the filename extension suggestion to `.rdfx` instead of `.rdfxml`